models/src/cloe/utility/frustum_culling_test.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright 2024 Robert Bosch GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * \file cloe/utility/frustum_culling_test.cpp
+ * \see  cloe/utility/frustum_culling.hpp
+ */
+
+#include <list>
+#include <tuple>
+
+#include <gtest/gtest.h>
+#include <cloe/utility/frustum_culling.hpp>
+
+TEST(rotate_point, rotate_point) {
```

Comment:
Name tests after what is being modelled. So... `TEST(models_frustum_culling, ...)` in all places here. This makes it easier to find test source.

Reply:
renamed.